docs/uninstall-an-operator.md

```diff
@@ -1 +1,36 @@
 # How to uninstall an Operator?
+
+In order to uninstall an operator, you need to delete the following resources:
+
+- Subscription
+- ClusterServiceVersion (CSV)
+
+Both `Subscription` and `ClusterServiceVersion` are namespace objects meaning you need to delete a `Subscription` and a `CSV` in a specific namespace where you install the operator into.
+
+## Delete a Subscription
+
+If you wish to look up a list of `Subscription` in a specific namespace to see which `Subscription` you want to delete, you can use the following `kubectl` command:
+```bash
+$ kubectl get subscription -n <namespace>
```

Comment:
Should we bother including a command to list subscriptions across all namespaces?

Reply:
Given `Subscription` is a namespaced object, I don't think it is necessary to list subscriptions across all namespaces. Plus, user may not have access to all namespaces so the `--all-namespaces` may even resolve into failure.

Reply:
I like your point about the permissions - marking this as resolved.